tests/test_plugin_loader.py

```
@@ -2724,36 +2728,50 @@ class TestPluginDirectories(unittest.TestCase):
     """Test cases for plugin directory discovery and creation."""
 
     @patch("os.makedirs")
-    @patch("mmrelay.plugin_loader.get_base_dir")
+    @patch("mmrelay.paths.get_home_dir")
+    @patch("mmrelay.paths.get_legacy_dirs")
     @patch("mmrelay.plugin_loader.get_app_path")
     @patch("mmrelay.plugin_loader.logger")
     def test_get_plugin_dirs_user_dir_success(
-        self, mock_logger, mock_get_app_path, mock_get_base_dir, mock_makedirs
+        self,
+        mock_logger,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_logger`

(ARG002)